accumulators/include/alps/accumulators/feature/max_num_binning.hpp

```diff
@@ -345,7 +345,7 @@ namespace alps {
                 void partition_bins(alps::mpi::communicator const & comm,
                                     std::vector<typename mean_type<B>::type> & local_bins,
                                     std::vector<typename mean_type<B>::type> & merged_bins,
-                                    int root) const
+                                    int /*root*/) const
```

Comment:
`partition_bins()` does not use `root` argument and it is not needed for any overload -- shall we eliminate it? (I guess it is here as a placeholder, if one day we decide to use `MPI_Reduce()` rather than `MPI_Allreduce()`...)

Reply:
Is it worth it? I mean the writing's on the wall anyway for `alps::accumulators`